cyberbrain-vsc/src/rpc_server.ts

```diff
@@ -28,8 +31,11 @@ export class RpcServer {
       // Sends data to the trace graph in webview.
       let webviewPanel = openTraceGraph(this.context);
       webviewPanel.webview.onDidReceiveMessage(
-        (message: string) => {
-          if (message === "Webview ready") {
+        (message: {
+          command: string;
+          context?: { interactionType: string; info?: any };
```

Comment:
Could you rename `context` to something else? There's another `this.context` in this method, which makes the code confusing to readers.